include/motor-control/core/stepper_motor/motor_interrupt_handler.hpp

```diff
@@ -380,13 +389,13 @@ class MotorInterruptHandler {
                 hardware.get_encoder_pulses();
 #ifdef USE_PRESSURE_MOVE
             if (buffered_move.sensor_id != can::ids::SensorId::UNUSED) {
-                auto msg = can::messages::BindSensorOutputRequest{
-                    .message_index = buffered_move.message_index,
-                    .sensor = can::ids::SensorType::pressure,
-                    .sensor_id = buffered_move.sensor_id,
-                    .binding = static_cast<uint8_t>(0x3)  // sync and report
-                };
-                send_to_pressure_sensor_queue(msg);
+                uint8_t binding = static_cast<uint8_t>(0x3);  // sync and report
```

Comment:
import and use the enum by name?

Reply:
This is a combo of multiple enums, and since the enum isn't a uint8_t it would have resulted in lots of casts and logic that would have confused it.